cmake/ReadoutCardDependencies.cmake

```diff
@@ -88,12 +88,16 @@ o2_define_bucket(
   ${DIM_INCLUDE_DIRS}
 )
 
+# Build the variable name for the path to boost python
+string(TOUPPER ${boost_python_component} upper)
```

Comment:
I think you should add if/else to this line as well, otherwise it will fail on mac as `${boost_python_component}` variable is empty.

Reply:
I had tested it and it works. `boost_python_component` exists and is an empty string. I guess that is fine with `TOUPPER`.

Reply:
All right, I think my CMake (`3.12.3`) is more strict or maybe the issue is somewhere else:
```
CMake Error at cmake/ReadoutCardDependencies.cmake:92 (string):
  string no output variable specified
```